lib/galaxy/webapps/tool_shed/api/repositories.py

```diff
@@ -88,7 +88,7 @@ def add_repository_registry_entry( self, trans, payload, **kwd ):
         return response_dict
 
     @web.expose_api_anonymous
-    def get_ordered_installable_revisions( self, trans, **kwd ):
+    def get_ordered_installable_revisions( self, trans, name, owner, **kwd ):
```

Comment:
I guess this should be `get_ordered_installable_revisions( self, trans, name = None, owner = None, **kwd )` ?